src/components/structural/header/ProjectView.js

```diff
@@ -423,6 +435,27 @@ class Project extends React.Component {
                                     }
                                     <span>&nbsp;</span>Preview
                                 </Button>
+                                <input
+                                    type="file"
+                                    accept="application/json"
+                                    ref={this.uploadRef}
+                                    onChange={(e) => this.props.importFunc(this.props.user.uid, e.target.files)}
```

Comment:
Also returns error when user is not logged in.

Reply:
Probably hide the option if user is not logged in (or import the scene without saving as a scene)